packages/components/psammead-image/src/index.jsx

```diff
@@ -15,6 +15,9 @@ export const Img = ({ alt, src, srcset, height, width }) => {
   if (srcset) {
     props.srcSet = srcset;
   }
+  if (!width) {
```

Comment:
We dont need this, if `width` is falsey, it wont have an affect on the component

Reply:
We need it in order to avoid `width=null` in the snapshot data
![](https://www.dropbox.com/s/9f6t3vbkyjz075m/width-null.png?dl=1)

Reply:
is that not what happens with height? React is clever enough to not render `null` in the dom

Reply:
I understand the React bit. I would like to get your opinion on this. Would you consider
```jsx
<img width={null} src="http://example.com/car.png"/>
```
to be a snapshot of 
```jsx
<img src="http://example.com/car.png"/>
```
That is actually rendered on the `DOM`.  IMO a snapshot should be what the component looks like when rendered.

Reply:
But if you didn't pass 'width=null' in the first place, it wouldn't show in the snapshot

Reply:
Since `width` is not a required prop what would the default value be?

Reply:
Null, we should just copy the approach taken with height, which didn't/doesnt need this logic

Reply:
The approach taken with `height` doesn't need it because there is no test defined for `image` without `height`. Going that approach means no tests for optional `width` too. But on the [issue description](https://github.com/bbc/psammead/issues/467#issue-436713861), it is stated
> There should be stories and unit test for the optional behaviour of the width prop.

Reply:
I agree we should have a test, and a snapshot with `width={null}` is fine :)